radio/src/audio_arm.cpp

```diff
@@ -135,6 +135,37 @@ const int16_t sineValues[] =
 };
 
 #if defined(SDCARD)
+
+const char * unitsFilenames[] = {
```

Comment:
reduce this array to only the unit string, i.e. "volt", "amp", then add 0 / 1 / 2 / 3 in the function, using `getStringWithIndex(...)`